README.md

```diff
@@ -17,5 +17,5 @@ components (no need to write them in the store file)
 
 ## Instructions
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_redux-list-of-todos/)
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://GGLUTT.github.io/react_redux-list-of-todos/)
```

Comment:
Add demo link to the PR description 
<img width="938" alt="image" src="https://github.com/user-attachments/assets/5bcdffc0-f3ac-4b45-818a-a8d2d9f74694">